src/classBased/components/Header.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const Header = () => {
+    const headerStyle = {
+        padding: "20px 0",
+        lineHeight: "1.5em",
+      }
+  return (
+    <header style={headerStyle}>
+        <h1
+            style={{
+                fontSize: "6rem",
+                fontWeight: "600",
+                marginBottom: "2rem",
+                lineHeight: "1em",
+                color: "#ececec",
+                textTransform: "lowercase",
+                textAlign: "center",
```

Comment:
[OPTIONAL] Kindly, move your CSS to a specific class everywhere in your project. remember that you can only put all your styling in a separate file and then import it in your code to use its rules. it's a good practice to improve performance and to clean up your code 😢

Reply:
Thanks for the feedback @mwafrika. I will put that into consideration